### PR TITLE
Upgrade to new IPSO Presence

### PR DESCRIPTION
**Pull request by thetoster**
Signed-off-by: bartlomiej.zarnowski <bartlomiej.zarnowski@imgtec.com>

### CODE REVIEW

File: project-conf.h [hunk 1]
```diff
@@ -37,10 +37,10 @@
 #define __PROJECT_LWM2M_CONF_H__
 
 /* Custom channel and PAN ID configuration for your project. */
-/*
-   #define RF_CHANNEL                  26
+
+   #define RF_CHANNEL                  25
    #define IEEE802154_CONF_PANID       0xABCD
```
Comment (abhijit-mahajani-imgtec) [inline project-conf.h]: please don't check in your developement values as defaults

File: symbols.c [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#include "symbols.h"
```
Comment (abhijit-mahajani-imgtec) [inline symbols.c]: symbols.c/h are not to be checked in

File: project-conf.h [hunk 3]
```diff
@@ -37,10 +37,10 @@
 #define __PROJECT_LWM2M_CONF_H__
 
 /* Custom channel and PAN ID configuration for your project. */
-/*
-   #define RF_CHANNEL                  26
+
+   #define RF_CHANNEL                  25
```
Comment (nikhil-zinjurde-imgtec) [inline project-conf.h]: This doesn't need changing.  Was it by mistake?

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,5 @@
+*.hex
+*.mikro-e
+*.map
+*.a
+obj_mikro-e/**/*.*
```
Comment (nikhil-zinjurde-imgtec) [inline .gitignore]: .gitignore seems to be checked in by mistake?

Comment (abhijit-mahajani-imgtec) [inline .gitignore]: it might be good to add .gitignore however for consistency need to be done at other clicker-apps repos as well. Any other opinions?

**Pull request closed without merging**